README.md

```diff
@@ -60,9 +60,9 @@ indexfile:
 
 ## RNA-seq and Exome-Seq figures
 
-The code to generate the omics figures in the manuscript can be found in the [figs](figs/) directory. Specifically the ExomeSeq markdown can be viewed [here]() and the RNASeq markdown can be viewed [here](). 
+The code to generate the omics figures in the manuscript can be found in the [figs](figs/) directory. Specifically the ExomeSeq markdown can be viewed [here]() and the RNASeq markdown can be viewed [here](). These notebooks are also contained in the R demo notebook container described below. 
 
```

Comment:
To regenerate the figure panel showing copy number variant analysis, refer to docker_cnv [here](https://github.com/Sage-Bionetworks/JHU-biobank/tree/master/figs/docker_cnv).

Reply:
is this docker container deposited anywhere?

Reply:
in nfosi

Reply:
OK, I will link to that instead.

Reply:
Sounds good :)

Reply:
OK, I've updated the docker repo for the cnv docker container to autobuild, so now any changes pushed to master will automatically be incorporated.

Reply:
whoops, wrong account :)